specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/recoverableManagedDatabases.json

```diff
@@ -0,0 +1,283 @@
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "List recoverable databases by managed instances": {
+            "$ref": "./examples/ListRecoverableManagedDatabasesByServer.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/managedInstances/{managedInstanceName}/recoverableDatabases/{recoverableDatabaseName}": {
+      "get": {
+        "tags": [
+          "RecoverableManagedDatabases"
+        ],
+        "description": "Gets a recoverable managed database.",
+        "operationId": "RecoverableManagedDatabases_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagedInstanceNameParameter"
+          },
+          {
+            "name": "recoverableDatabaseName",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the specified recoverable managed database.",
+            "schema": {
+              "$ref": "#/definitions/RecoverableManagedDatabase"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidRecoverableDatabaseId - Invalid recoverable database identifier.\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-examples": {
+          "Gets a recoverable databases by managed instances": {
+            "$ref": "./examples/GetRecoverableManagedDatabase.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RecoverableManagedDatabaseListResult": {
+      "description": "A list of recoverable managed databases.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RecoverableManagedDatabase"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "RecoverableManagedDatabaseProperties": {
+      "description": "The recoverable managed database's properties.",
+      "type": "object",
+      "properties": {
+        "lastAvailableBackupDate": {
```

Comment:
This should be ISO 8601 compliant and so the format should be date-time

Reply:
This format is used for restore workflow . Customer don't need change any format when pass in pointintime.

Reply:
This format is used for restore workflow . Customer don't need change any format when pass in pointintime.

Reply:
Lixia, what should this property's data type be in generated C# code? string, or datetime?